generators/entity-server/templates/src/main/java/package/web/rest/EntityResource.java.ejs

```diff
@@ -58,6 +58,7 @@ import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 <%_ } _%>
 import org.springframework.http.ResponseEntity;
+import org.springframework.transaction.annotation.Transactional;
```

Comment:
Transactional is only used for SQL databases, so add the EJS condition `<%_ if (databaseType === 'sql') { _%>`